docs/cli/geo/maps.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Maps
+description: Use Amplify CLI to create and manage maps to visualize GeoSpatial data in your app.
```

Comment:
GeoSpatial data is not fully understandable and also not has weird casing.

Reply:
changed to geospatial